docs/source/en/training.mdx

```diff
@@ -151,66 +151,112 @@ Then fine-tune your model by calling [`~transformers.Trainer.train`]:
 
 <Youtube id="rnTGBy2ax1c"/>
 
+## Train with Keras
+
 🤗 Transformers models also supports training in TensorFlow with the Keras API.
 
-### Convert dataset to TensorFlow format
+### Loading data for Keras
+
+When you want to train a 🤗 Transformers model with the Keras API, you need to convert your dataset to a format that
+Keras understands. If your dataset is small, you can just convert the whole thing to NumPy arrays and pass it to Keras.
+
+First, load a dataset:
+
+```py
+from datasets import load_dataset
+
+dataset = load_dataset("glue", "cola")  # Simple text classification dataset
+dataset = dataset["train"]  # Just take the training split for now
```

Comment:
Use the prose to explain what you're doing instead of comments in the code.

Reply:
Done!